tests/queries/0_stateless/03215_multilinestring_geometry.sql

```diff
@@ -0,0 +1,12 @@
+-- { echoOn }
```

Comment:
Need at least one query with non-constant input.

Reply:
(better with different array sizes of both the number of lines and the number of points)